modules/local/convert_ont_read_names.nf

```diff
@@ -25,6 +25,18 @@ process CONVERT_ONT_READ_NAMES {
                     args.contains("--output-fmt cram") ? "cram" :
                     input.getExtension()
     if ("$input" == "${prefix}.${file_type}") error "Input and output names are the same, use \"task.ext.prefix\" to disambiguate!"
+
+    // The SED looks for readnames in a BAM file
+    // and replaces all hyphens - with underscores _,
+    // then it adds /77923358/ccs/ to the read name
+    // to make ONT read names look like they come from PacBio.
```

Comment:
Interesting, for which programs are this required?

Reply:
TRGT. It might have changed in more recent updates, I will look into it when we #47.